recipes/clarabel/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "clarabel" %}
+{% set version = "0.6.0" %}
+{% set checksum = "ef909a393e72981ca10b1d866d9cc7fb6295ece20ae035def764338894961184" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.python.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz  
+  sha256: {{ checksum }}
```

No need for a variable that will be used only once. This just obfuscates the recipe.

```suggestion
  sha256: ef909a393e72981ca10b1d866d9cc7fb6295ece20ae035def764338894961184
```